### ISSUE TITLE
JOSS Review 2 

### ISSUE THREAD

**Issue by harisorgn** [lsandig/Kirstine.jl#4]
Overall I found `Kirstine.jl` a great contribution to experimental design and to the Julia ecosystem. The motivation is clear and convincing, the codebase well organised and thoroughly tested and the documentation is extensive. 

I am covered by past comments in #2  and #3 and am happy to sign off this article for publication. Only have a couple of typos for @lsandig consideration :

- minor typo, `1` and `2` superscripts should be flipped here to make the explanation after the equation less confusing :
https://github.com/lsandig/Kirstine.jl/blob/1e23cd0d1c98c3ba209b0a41a7b5acde4b3e3251/docs/src/math.md?plain=1#L428

- another (more) minor typo, should be `t_{\DimUnit}` instead of `1_...` here :
https://github.com/lsandig/Kirstine.jl/blob/1e23cd0d1c98c3ba209b0a41a7b5acde4b3e3251/docs/src/dtr.md?plain=1#L62

**Comment by lsandig** [comment]
> minor typo, `1` and `2` superscripts should be flipped here to make the explanation after the equation less confusing :

Actually, I'd like to keep the superscripts in the order they are in. Flipping them means that I would have to also flip the arguments in the definition of relative efficiency to `RelEff(ζ_2, ζ_1)`, which I consider to be more confusing. (Especially since some authors in the literature define a one-argument relative efficiency where `ζ_2` is implicitly set to the solution of the design problem. I prefer this "optional" argument to come second.)

Alternatively, one could define relative efficiency as the reciprocal of what it is now (in the Fedorov/Leonov book this reciprocal quantity goes by the name of relative _de_ficiency), but then it would not be bounded when one of the design measures is optimal.

So, I kept the notation and definition, but tried to make the interpretation a bit clearer: https://git.sr.ht/~lsandig/Kirstine.jl/commit/d5b556b

> another (more) minor typo, should be `t_{\DimUnit}` instead of `1_...` here

Fixed. https://git.sr.ht/~lsandig/Kirstine.jl/commit/2490ddd
